leads/lead_monitoring.md

```diff
@@ -63,6 +63,50 @@ These are just a few of the integrations already created by the Mautic community
 
 **Note:** It is important to note that you are not limited by these plugins and you can place the tracking pixel directly on any HTML page for website tracking.
 
+### Mobile Monitoring
+
+The essence of monitoring what happens in an App is similar to monitoring what happens on a website. Mautic contains the building blocks needed for native (or pseudo-native) and HTML5-wrapper based Apps, regardless of platform.
+
+In short, we use named screen views (e.g. main_screen) in the App as our page_url field in the tracker, and the lead's email as the unique identifier, like this:
```

Comment:
This formulation is sort of misleading. People reading the docs would have no idea who "we" are.